### ISSUE TITLE
yolo v8 training on 50 epochs with train\box loss = 1.2

### ISSUE THREAD

**Issue by puneet2059** [chiragsamal/Steel-Surface-Defect-Detection#2]
I am right now training my model with 2900 images 416*416 and I am getting metric\precision as 92%, metric\precision as 77 %,metric\map50 =87%  in 50 epochs with train\box loss = 1.2  but when my confusion matrix is built I get a 100 % false detection rate which is kind of issue for me I tried altering the parameters by conf=0.5,iou= 0.7,nms=0.85,augmentation= True. But still no effect on false positives.
I need help on this one. I have small-size defects if you have any particular model in mind also suggest  that.
